src/main/java/uk/gov/hmcts/cmc/claimstore/rules/MoreTimeRequestRule.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.cmc.claimstore.rules;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.exceptions.MoreTimeAlreadyRequestedException;
+import uk.gov.hmcts.cmc.claimstore.exceptions.MoreTimeRequestedAfterDeadlineException;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import java.time.LocalDate;
+import java.util.Objects;
+import javax.validation.constraints.NotNull;
+
+@Service
+public class MoreTimeRequestRule {
+
+    public void assertMoreTimeCanBeRequested(@NotNull Claim claim) {
```

Comment:
The `@NotNull` annotation is not being used here. If that was one of the annotation supported by IntelliJ (https://www.jetbrains.com/help/idea/nullable-and-notnull-annotations.html#notnull) then we would benefit from it but that one is useless. Unfortunately these JetBrains annotations are not combined into byte code (by default).

At some point we could evaluate useless of checker framework (see: https://checkerframework.org)

Please remove the annotation.

Reply:
Removed.